src/plugins/dashboard/public/application/utils/use/use_saved_dashboard_instance.ts

```diff
@@ -90,7 +88,9 @@ export const useSavedDashboardInstance = (
         }
 
         setSavedDashboardInstance(savedDashboard);
-      } catch (error) {}
+      } catch (error) {
+        console.log(error);
```

Comment:
remove?

Reply:
Will use toast notification to throw error here